<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartWidget</class>
 <widget class="FloatingWidget" name="ChartWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="unicodeCoverageStat">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Coverage of Unicode Interval</string>
       </property>
       <property name="text">
        <string>Block:</string>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
       <property name="buddy">
        <cstring>uniPlaneCombo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="uniPlaneCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Search:</string>
       </property>
       <property name="buddy">
        <cstring>charSearchLine</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="charSearchLine">
       <property name="toolTip">
        <string>Search a character by its codepoint (expressed in hexadecimal 
and prefixed with &quot;U+&quot;, &quot;u+&quot; or &quot;+&quot;) or a glyph by its name.
Press enter after editing to run the search.</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="FMGlyphsView" name="abcView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="statusTip">
      <string>Clicking on a glyph will show details about it</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="rubberBandSelectionMode">
      <enum>Qt::IntersectsItemBoundingRect</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QToolButton" name="clipboardCheck">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Set Mode &quot;Copy Character to Clipboard&quot;</string>
       </property>
       <property name="text">
        <string>Copy Mode</string>
       </property>
       <property name="icon">
        <iconset resource="application.qrc">
         <normaloff>:/action-copy-char</normaloff>:/action-copy-char</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="uniLine"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="FloatingWidgetToolBar" name="toolbar" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FloatingWidget</class>
   <extends>QWidget</extends>
   <header>floatingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FloatingWidgetToolBar</class>
   <extends>QWidget</extends>
   <header>floatingwidgettoolbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FMGlyphsView</class>
   <extends>QGraphicsView</extends>
   <header>fmglyphsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
